client/tsconfig.json

```diff
@@ -18,8 +18,8 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "noUnusedLocals": true,
```

Comment:
unused parameters and arguments make code a lot harder to understand. please undo this change